docs/zarr-v3.md

```diff
@@ -0,0 +1,96 @@
+# zarr-v3 Guide/Roadmap
+
+`anndata` now uses the much improved {mod}`zarr` v3 package and also allows writing of datasets in the [v3 format], with the exception of structured arrays.
+Users should notice a significant performance improvement, especially for cloud data, but also likely for local data as well.
+Here is a quick guide on some of our learnings so far:
+
+## Remote data
+
+We now provide the {func}`anndata.experimental.read_lazy` feature for reading as much of the {class}`~anndata.AnnData` object as lazily as possible, using `dask` and {mod}`xarray`.
+Please note that this feature is experimental and subject to change.
+To enable this functionality in a performant and feature-complete way for remote data sources, we use [consolidated metadata] on the `zarr` store (written by default).
+Please note that this introduces consistency issues - if you update the structure of the underlying `zarr` store i.e., remove a column from `obs`, the consolidated metadata will no longer be valid.
+Further, note that without consolidated metadata, we cannot guarantee your stored `AnnData` object will be fully readable.
+And even if it is fully readable, it will almost certainly be much slower to read.
+
+There are two ways of opening remote [`zarr` stores] from the `zarr-python` package, `fsspec` and `obstore`, and both can be used with `read_lazy`.
+[`obstore` claims] to be more performant out-of-the-box, but notes that this claim has not been benchmarked with the `uvloop` event loop, which itself claims to be 2X more performant than the default event loop for `python`.
```

Comment:
This paragraph is a a bit nested, maybe more clearly itemize the options: Are they `fsspec`, `fsspec`+`uvloop`, `obstore`, or … ?

Also:

```suggestion
[`obstore` claims] to be more performant out-of-the-box, but notes that this claim has not been benchmarked with the `uvloop` event loop, which itself claims to be 2× more performant than the default event loop for `python`.
```